src/GUI/ccextractorGUI.c

```diff
@@ -260,31 +279,35 @@ int main(void)
 	media.icons.img_file = icon_load("../../icon/img.png");
 	media.icons.movie_file = icon_load("../../icon/movie.png");
 #else
-    media.icons.home = icon_load("../icon/home.png");
-    media.icons.directory = icon_load("../icon/directory.png");
-    media.icons.computer = icon_load("../icon/computer.png");
-    media.icons.desktop = icon_load("../icon/desktop.png");
-    media.icons.default_file = icon_load("../icon/default.png");
-    media.icons.text_file = icon_load("../icon/text.png");
-    media.icons.music_file = icon_load("../icon/music.png");
-    media.icons.font_file =  icon_load("../icon/font.png");
-    media.icons.img_file = icon_load("../icon/img.png");
-    media.icons.movie_file = icon_load("../icon/movie.png");
+	const char* path_of_executable = get_executable_path();
+	char path_of_ccextractor[strlen(path_of_executable)-20]; // This is the size of: mac/./ccextractorGUI and linux/ccextractorGUI
+	strncpy(path_of_ccextractor, path_of_executable, strlen(path_of_executable));
+	path_of_ccextractor[strlen(path_of_ccextractor)-20]='\0'; // Fails without null pointer
+	media.icons.home = icon_load(concat(path_of_ccextractor,"icon/home.png"));
+	media.icons.directory = icon_load(concat(path_of_ccextractor, "icon/directory.png"));
```

Comment:
This is leaking memory - concat is doing a malloc() but there's no free() anywhere, and you're not holding a reference. 
A simple solution would be to write a function called cc_icon_load (or a better name you come up with) that just takes the name of the icon ("home") for example and adds everything else (path_of_executable) + ".png", then frees everything, and just returns the same same as icon_load().
A *much better* solution would be to write a simple program that takes all the icons and generates a .h file that contains the icons themselves (not the filenames; the binary data) in an array so we can generate a binary and contains the icons completely - so we don't have to have any additional file.

Reply:
While working on the better solution stbi uses fopen to read the file so unless we rewrite the library the function is the better method won't work in the situation